crates/sui-graphql-e2e-tests/tests/epoch/epoch.exp

```diff
@@ -50,11 +50,14 @@ Response: {
               "name": "validator-0"
             }
           ]
-        }
+        },
+        "validatorCandidatesSize": 0,
+        "inactivePoolsId": "0x3e79098bee2cf7d5cdd6e31a71f608f22b90af2952ce19c031910186a5c9cca8"
       },
       "totalGasFees": "1000000",
       "totalStakeRewards": "1000000",
       "totalStakeSubsidies": "0",
+      "totalTransactions": "7",
```

Comment:
Is this actually 7 txns? Even if we count the number of commands, I don't think we get 7... reviewers please help me check.

Reply:
It's very hard to say -- can you also query the transactionBlocks in the epoch?  Maybe just query for the transaction kind and the digest?

Reply:
Looking at how this is implemented, if this is a query about the latest epoch, then there may be a race here (also let's add a test for querying both a historical epoch and the current epoch).

Reply:
Good idea. It's not the latest epoch though.

Reply:
I think there's indeed some race going on, with transaction blocks it seems. `transactionBlocks` is empty, it shouldn't be. And `totalTransactions` is sometime 3, sometimes 7. 3 would be the correct answer and 7 seems to be the total transactions that have ever happened so maybe there's somewhere where we take the network total directly? @amnn @wlmyng

Reply:
As per our out-of-band discussion, I suspect this is an issue on the write side, given the epoch in question is not the latest one, so I think we should be okay to land this change, and separately investigate the source of this issue.